docker-compose-tests.yml

```
@@ -10,13 +10,12 @@ services:
         GITHUB_USERNAME: "${GITHUB_USERNAME}"
         GITHUB_TOKEN: "${GITHUB_TOKEN}"
     environment:
-      - NEO4J_TEST_URL=http://neo4j:7474
-      - NEO4J_BOLT_TEST_URL=bolt://neo4j:7687
+      - NEO4J_TEST_URL=bolt://neo4j:7687
     command: [ "go", "test", "-mod=readonly", "-tags=integration" , "./..." ]
```

`suggestion`: Can we add the verbose flag please.
```suggestion
    command: [ "go", "test", "-v", "-mod=readonly", "-tags=integration" , "./..." ]
```